spec/features/events_spec.rb

```diff
@@ -72,7 +72,7 @@
   end
 
   scenario "logged in as Organizer I want to be able to send an email to all accepted applicants" do
-    @event = FactoryGirl.create(:event, :in_selection_phase)
+    @event = FactoryGirl.create(:event, :in_selection_phase, acceptances_have_been_sent: false)
```

Comment:
That the value gets changed is already ensured in a different test :) But skipping sounds nice